WooCommerce/Classes/ViewRelated/Orders/OrderDetailsViewController.swift

```diff
@@ -431,6 +485,20 @@ private extension OrderDetailsViewController {
         }
     }
 
+    func configureTracking(cell: OrderTrackingTableViewCell, at indexPath: IndexPath) {
+        guard let tracking = orderTracking(at: indexPath) else {
+            return
+        }
+
+        cell.topText = tracking.trackingProvider
+        cell.bottomText = tracking.trackingNumber
+        cell.actionButtonNormalText = viewModel.trackTitle
+
+        cell.onActionTouchUp = { [ weak self ] in
+            self?.trackWasPressed(indexPath.row)
```

Comment:
I would name this `trackingWasPressed` because it might be confusing if this result pops up when searching for `track` events in the code.